SourceKitStressTester/Tests/StressTesterToolTests/StressTesterToolTests.swift

```diff
@@ -10,69 +10,110 @@
 //
 //===----------------------------------------------------------------------===//
 
-import XCTest
 import Common
 import StressTester
+import XCTest
 
 class StressTesterToolTests: XCTestCase {
   var workspace: URL!
-
-  var stressTesterPath: String!
-  var testFilePath: String!
+  var testFile: URL!
+  var swiftcPath: String!
 
   func testCommandLine() {
```

Comment:
What are your thoughts on these tests? Could also check for invalid swiftc/tempDir